### PR TITLE
feat: Update Symfony for Scoring and Reports applications

### PR DESCRIPTION
**Pull request by mccar**
# Pull request summary

Related to [REL-1724](https://oat-sa.atlassian.net/browse/REL-1724).

Update Symfony to 6.4

# How to test

Deployed to unit05. Check test cases.

# Related PRs
https://github.com/oat-sa/tao-manual-scoring-be/pull/645
https://github.com/oat-sa/tao-scoring-service-be/pull/374
https://github.com/oat-sa/tao-hbl-scoring-engine-be/pull/114
https://github.com/oat-sa/tao-simple-reports-be/pull/196

[REL-1724]: https://oat-sa.atlassian.net/browse/REL-1724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
-        "oat-sa/lib-lti1p3-core": "^7.0",
+        "oat-sa/lib-lti1p3-core": "^7.0.2",
         "psr/log": "^1 || ^2 || ^3",
-        "symfony/framework-bundle": "^5.4",
-        "symfony/psr-http-message-bridge": "^2.0",
-        "symfony/security-bundle": "^5.4",
-        "symfony/security-core": "^5.4",
-        "symfony/security-http": "^5.4",
-        "symfony/yaml": "^5.4"
+        "symfony/framework-bundle": "^6.0",
```
Comment (yaraslau-kavaliou) [inline composer.json]: Does it make sense to here `^6.4`, wdyt (here and in all other places with 6.0)

**Review by yaraslau-kavaliou — APPROVED** [review]

File: Security/Firewall/Message/LtiPlatformMessageAuthenticator.php [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    private FirewallMap $firewallMap;
+
+    private PlatformLaunchValidatorInterface $validator;
+
+    private string $firewallName;
+
+    private array $types;
+
+    public function __construct(
+        FirewallMap $firewallMap,
```
Comment (yaraslau-kavaliou) [inline Security/Firewall/Message/LtiPlatformMessageAuthenticator.php]: suggestion: If we user php >= 8 maybe it makes sense to use https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion for new classes?

**Review by yaraslau-kavaliou — APPROVED** [review]
- [ ]  New code is covered by tests
- [x]  Tests are running successfully on my local machine
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  Feature is working correctly on my local machine
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful
- [ ]  Pull request's target is not master
- [ ]  Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ]  Commits messages are meaningful
- [ ]  Commits are atomic

File: Security/Firewall/Message/LtiPlatformMessageAuthenticator.php (hunk 2 above)
Comment (mccar) [reply Security/Firewall/Message/LtiPlatformMessageAuthenticator.php]: fixed

**Pull request merged**